README.md

```diff
@@ -32,20 +32,50 @@ Installation (Install Script)
 
 Requirements
 ------------
-* Windows 10 1803, 1809, 1903, 1909, or 2004
+* Windows 10 1803, 1809, 1903, 1909, 2004, 20H2, or 21H1
 > Insider Preview editions of Windows are not supported
 * 60 GB Hard Drive
 * 2 GB RAM
 
 Recommended
 -----------
-* Windows 10 2004
+* Windows 10 21H1
 * 80+ GB Hard Drive
 * 4+ GB RAM
 * 2 network adapters
 
 Instructions
 ============
+Pre-Install Procedures
+----------------------
+You MUST disable Windows Defender for a smooth install. The best way to accomplish this is through Group Policy
```

Comment:
Should we mention something for people who can't use Group Policy? For instance, those using free Windows development VMs? I think Group Policy is only available for Professional, Enterprise, and Education editions of Windows.

I've found some resources mentioning ways to enable group policy using third party applications (kinda sketchy):
- https://www.majorgeeks.com/content/page/enable_group_policy_editor_in_windows_10_home_edition.html

At a minimum, we could reference / link to these methods and mention upfront it's sketchy and may be malware since we're not going to verify / make any claims otherwise.

Reply:
I think we can leave the sketchy stuff out. I checked the Windows 10 Dev download page and it is indeed Enterprise edition, so there should be no problems with group policy for all of the free users.

Reply:
Awesome! Thanks for checking